tools/depends/target/libcdio-gplv3/Makefile

```diff
@@ -23,7 +23,9 @@ all: .installed-$(PLATFORM)
 $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE).$(HASH_TYPE) $(DEPS)
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
+ifeq ($(TARGET_PLATFORM),macosx)
```

Comment:
Why gate the patch? It has no effect on any other platform

Reply:
Builds failed for iOS and tvos without it, see:
https://jenkins.kodi.tv/job/TVOS/12341/
https://jenkins.kodi.tv/job/IOS-ARM64/21337/

Reply:
Add exclusion to https://github.com/xbmc/xbmc/blob/57dd0d24260ee84d56731c69b02615a4e5b8b6b9/tools/depends/target/Makefile#L60 for libcdio-gplv3

Reply:
Just stick with the if. its probably going to open a can of worms i dont want to deal with right now.